components/applicationsets/ztp-clusters-appset.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: clusters-sub
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: clusters
+  namespace: openshift-gitops
+spec:
+  generators:
+    - git:
+        repoURL: https://github.com/redhat-partner-solutions/vse-carslab-hub.git
+        revision: main
+        directories:
+          - path: clusters/ztp-overlays/*
```

Comment:
Is this path going to trigger a deployment? I see a `SiteConfig` folder inside skidoo folder